src/nvm/file.rs

```diff
@@ -131,7 +215,7 @@ impl FileNvmBuilder {
         // OpenOptions::createはファイルが既に存在する場合はそれを開き
         // 存在しない場合は作成する
         options.create(true);
-        let file = track_io!(options.open(&filepath))?;
+        let file = self.file_open_with_error_info(true, &options, &filepath)?;
```

Comment:
Please use `track(...)!` macro at here and other points at where `file_open_with_error_info` method is called.